dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml

```diff
@@ -1081,19 +1084,19 @@ paths:
                               example: null
     options:
       tags:
-        - Eligibility Screening Endpoints
+        - Eligibility
       description: >
-        The /eligibilityPrograms OPTIONS method returns a list of supported HTTP methods, a readable schema and an example request payload.
+        The OPTIONS method will return the HTTP methods that the server supports for the specified URL along with a readable schema and an example of the request payload.
```

Comment:
Did we still want to keep the options method?

Reply:
idk, it doesn't hurt? But maybe it's confusing and hopefully our documentation is good enough that we don't need an OPTIONS method : p

I'll remove the documentation for it for now.